src/laboratory-resource.ts

```diff
@@ -83,3 +84,7 @@ export function rejectLabOrder(orderId: string, comment: string, abortController
     },
   });
 }
+
+export const labDateRange = getGlobalStore<{ dateRange: Date[] }>('lab-date-range', {
```

Comment:
```suggestion
export const labDateRange = getGlobalStore<{ dateRange: Date[] }>('lab-order-date-range', {
```

Reply:
Hey @pirupius I am not sure what value we gain from renaming `lab-date-range` to `lab-order-date-range` , isn't the prior name clearer as it matches the variable name and also I do not really understand the `order` thing. Please help me understand.

Reply:
@amosmachora i don't think this suggestion will matter based on the other comments regarding the the storage mechanism. 
The suggestion was just to make it more specific. The purpose was specific to filtering lab orders and that's why i had suggested it.

Reply:
Ooh okay. Thanks for clearing that up.